composer.json

```diff
@@ -53,7 +53,8 @@
         "symfony/polyfill-php73": "^1.11",
         "symfony/polyfill-php80": "^1.15",
         "symfony/polyfill-uuid": "^1.15",
-        "symfony/runtime": "self.version"
+        "symfony/runtime": "self.version",
+        "vimeo/psalm": "4.x-dev"
```

Comment:
This is not needed.

Reply:
oops, leftovers from psalm local debug :) reverted